src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTask.java

```diff
@@ -236,7 +237,7 @@ private ZipFileProcessingResult processZipFileContent(
 
             Optional<OcrValidationWarnings> ocrValidationWarnings = this.ocrValidator.assertOcrDataIsValid(envelope);
 
-            Envelope dbEnvelope = toDbEnvelope(envelope, containerName, ocrValidationWarnings);
+            dbEnvelope = toDbEnvelope(envelope, containerName, ocrValidationWarnings);
 
             result.setEnvelope(envelopeProcessor.saveEnvelope(dbEnvelope));
```

Comment:
the only thing that is used from `dbEnvelope` (which is passed to `handleEventRelatedError`) is its ID, but is it ever set in case of an error?

Reply:
probably not, you are right. i'll review this